tools/data_builder.py

```diff
@@ -194,11 +220,15 @@ def parse_state(state, text):
             # Text without a markdown marker, this might be the description or metadata
             id_prefix = "id:"
             tags_prefix = "tags:"
+            lat_long_prefix = "geolocation:"
             if line.startswith(id_prefix):
                 entry["id"] = line[len(id_prefix) :].strip()
             elif line.startswith(tags_prefix):
                 spacey_tags = line[len(tags_prefix) :].split(",")
                 entry["tags"] = [tag.strip() for tag in spacey_tags]
+            elif line.startswith(lat_long_prefix):
+                entry["geolocation"] = validate_geo(line[len(lat_long_prefix) :].strip())
```

Comment:
The .strip() call here seems like a duplicate effort of the .strip() call on line 134, see if you can remove one (prefer removing the one here within method params)

Reply:
It's not quite duplicate because you're first stripping "geolocation: abc" then " abc", but I'll change it to an lstrip